ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -131,38 +131,14 @@
     <Compile Include="Animated\ValueAnimatedNode.cs" />
     <Compile Include="Bridge\BridgeBusyEventArgs.cs" />
     <Compile Include="Bridge\BridgeIdleEventArgs.cs" />
-    <Compile Include="Bridge\IAsyncCancelable.cs" />
-    <Compile Include="Bridge\IAsyncDisposable.cs" />
-    <Compile Include="Bridge\ILifecycleEventListener.cs" />
-    <Compile Include="Bridge\IPromise.cs" />
     <Compile Include="Bridge\ReactInstance.cs" />
-    <Compile Include="Bridge\CompiledReactDelegateFactory.cs" />
     <Compile Include="Bridge\DispatcherHelpers.cs" />
```

Comment:
I don't see where this shows up in ReactNative.Shared, but it is added back below at the root directory level for ReactNative.csproj,